projects/plugins/jetpack/extensions/blocks/like/like.php

```diff
@@ -31,21 +31,72 @@ function register_block() {
 /**
  * Like block render function.
  *
- * @param array $attr    Array containing the Like block attributes.
+ * @param array  $attr Array containing the Like block attributes.
+ * @param string $content String containing the Like block content.
+ * @param object $block Object containing the Like block data.
  *
  * @return string
  */
-function render_block( $attr ) {
+function render_block( $attr, $content, $block ) {
 	/*
 	 * Enqueue necessary scripts and styles.
 	 */
 	Jetpack_Gutenberg::load_assets_as_required( __DIR__ );
 
-	$output = 'This is where the like button will go.';
+	$html = '';
+
+	$uniqid  = uniqid();
+	$post_id = $block->context['postId'];
+	$title   = esc_html__( 'Like or Reblog', 'jetpack' );
+
+	/**
+	 * Enable an alternate Likes layout.
+	 *
+	 * @since 12.9
+	 *
+	 * @module likes
+	 *
+	 * @param bool $new_layout Enable the new Likes layout. False by default.
```

Comment:
There's an error in this doc line. It's true by default.
My bad - we can fix this in a follow-up PR.